packages/ui/src/panelup.ts

```diff
@@ -54,13 +54,13 @@ export function openPanel (
 }
 
 export function closePanel (shoulRedirect: boolean = true): void {
+  currentLocation = undefined
```

Comment:
Any side effect of this change?

Reply:
Yeah, it was broken in this scenario:
1. Open document with mention of a task
2. Click mention -> task opened
3. Click back -> document opened
4. Click mention again -> task NOT opened